docs/node/node.md

```diff
@@ -11,6 +11,11 @@ Bitcoin-s has experimental support for neutrino which is a new lite client propo
 read more about how neutrino works [here](https://suredbits.com/neutrino-what-is-it-and-why-we-need-it/). At this time, 
 bitcoin-s only supports connecting to one trusted peer.
 
+#### Limitations
```

Comment:
We should probably note this in the wallet docs too? Something like 

"Due to limitations in the node project, the wallet does not support _inbound_ unconfirmed transactions" 

We should be able to display outbound unconfirmed transactions as we have those locally?